include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -110,15 +117,17 @@ namespace parallel
        *
        * @note Only used for 2D and 3D.
        */
-      std::array<types::material_id, GeometryInfo<dim>::lines_per_cell>
+      std::array<types::material_id,
+                 dim == 1 ? 0 : GeometryInfo<dim>::lines_per_cell>
```

Comment:
I guess this makes sense. You'd need something like `lines_per_cell_on_the_boundary`, but that doesn't exist. Is that the issue?

Reply:
Yes.